### ISSUE TITLE
No information about generating bearer tokens

### ISSUE THREAD

**Issue by piotrek-k** [aspnet/Docs#9940]
Thanks for this post! Really useful. But, although steps for getting and accepting incoming tokens are provided, I can't find any information about how should I generate such token. It would be useful if you added some examples or at least "Additional Resources" below on how to configure it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d0da2144-3730-f8ef-1d16-a89f11fd6198
* Version Independent ID: 6d8f5bc8-8f5c-ba13-b7f2-0366eac46f81
* Content: [Authentication and authorization in ASP.NET Core SignalR](https://docs.microsoft.com/en-us/aspnet/core/signalr/authn-and-authz?view=aspnetcore-2.2)
* Content Source: [aspnetcore/signalr/authn-and-authz.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/signalr/authn-and-authz.md)
* Product: **aspnet-core**
* GitHub Login: @tdykstra
* Microsoft Alias: **anurse**

**Comment by tdykstra** [comment]
@piotrek-k Good point, that's a gap in our documentation.  The best we have for now that I'm aware of is  this blog post: https://blogs.msdn.microsoft.com/webdev/2016/10/27/bearer-token-authentication-in-asp-net-core/
